### ISSUE TITLE
Elements are not Accessible with the Keyboard #UX #Ergonomics #Accessibiliy

### ISSUE THREAD

**Issue by OLLI-S** [bitwarden/web#712]
## Describe the Bug
In the Web Vault I checked the tab order and if all elements are accessible with the TAB key.

### Unlock Vault (enter master password)
Here all elements are accessible with the TAB key, **no issues were found**.

### My Vault
Here all elements are accessible with the TAB key, **no issues were found**.

### Adding / Editing an entry

![image](https://user-images.githubusercontent.com/8037790/99903252-94b3fa80-2cc3-11eb-8f33-fd92e57d019f.png)

The elements marked with a red border are not accessible with the TAB key.
Number 5 and number 6 focus the generate and check icons above the field.

At custom password fields also the buttons to see the password and to copy it to the clipboard are not accessible using the TAB key:

![image](https://user-images.githubusercontent.com/8037790/99903363-31769800-2cc4-11eb-98d3-8a31bb60aeb3.png)

I know that I can copy the content of a field using CTRL+C, but nevertheless all UI-elements that are visible, enabled and interactive must be accessible with the keyboard.

### Tools
In the Tools I checked every single page, but also here all elements are accessible with the TAB key, **no issues were found**.

### Settings
Also in the Settings I checked every single page (I did **not yet check the Organizations**). 
Also here all elements are accessible with the TAB key, **no issues were found**.

## Steps To Reproduce
Just use the TAB key to cycle through the elements

## Expected Result
All interactive elements must be accessible with the keyboard (according to WCAG 2.1, chapters [2.1.1](https://www.w3.org/TR/WCAG21/#keyboard) and [2.1.2](https://www.w3.org/TR/WCAG21/#no-keyboard-trap))

## Environment
- Operating system: Windows 10 Pro 64-Bit Version 20H2 (Build 19042.630)
- Browser: Microsoft Edge Chromium 87.0.664.41 (Offizielles Build) (64-Bit)
- Build Version: 2.17.0

**Comment by eliykat** [comment]
Thanks for taking the time to audit this and report the issue, @OLLI-S!
